addons/l10n_ar/models/res_partner.py

```diff
@@ -64,7 +64,7 @@ def check_vat(self):
         # NOTE by the moment we include the CUIT (VAT AR) validation also here because we extend the messages
         # errors to be more friendly to the user. In a future when Odoo improve the base_vat message errors
         # we can change this method and use the base_vat.check_vat_ar method.s
-        l10n_ar_partners = self.filtered(lambda x: x.l10n_latam_identification_type_id.l10n_ar_afip_code)
+        l10n_ar_partners = self.filtered('l10n_latam_identification_type_id')
```

Comment:
What you do here does not make any sense to me.  We need the filtering on the AFIP code simply already to identify it is an Argentinean identification type.  And I don't see why it would not have worked, because for CUIT, the code is 80.

Reply:
I just updated it.

Reply:
@yosuanicolaus @jco-odoo not working after the latest changes were done, check my video below:
https://www.loom.com/share/b3a889acdc9d40989a3fd0138750bad8?sid=223561e3-a0f8-4052-a559-417bc75e99d8

Reply:
I just updated it again. Thanks for testing @vbe-odoo, you can test them again once the runbot is ready :)

It seems that whenever that error is originated from the base test towards the partner's VAT. And actually, it's not saying that the CUIT in the view is invalid, but rather it checks the VAT of the partner, and since the VAT label for Argentina is CUIT, it refers to it as CUIT.

With the new changes I made it so that if the partner have either a argentinan VAT (CUIT/DNI/CUIL) OR their country code is AR, we will not extend the test to the base test.

Reply:
Thanks for the fix @yosuanicolaus I've tested and now it works properly.
Ready for your review @jco-odoo